api/handler/get-survey.js

```diff
@@ -52,12 +52,10 @@ function getSurvey (request, reply) {
                 ON jsq.surveyTemplateId = st.id
                 JOIN question_template qt
                 ON qt.id = jsq.questionTemplateId
-                JOIN join_questions_and_options jqo
```

Comment:
Handled the change in process-survey-instance helper file.
Will test and provide the results.